app/src/main/res/layout/test_bottom_sheet_options_menu.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Does this need to be a constraint layout or could we just make the ImageButton top-level?

Reply:
Resolved by the file being removed.